website/source/docs/platform/k8s/servers-outside-kubernetes.html.md

```diff
@@ -52,3 +52,126 @@ and federating it with your cluster outside Kubernetes.
 You may also consider adopting Consul Enterprise for
 [network segments](/docs/enterprise/network-segments/index.html).
 
+## Configuring TLS with Auto-encrypt
+
+Consul's auto-encrypt feature allows clients to automatically provision their own certificates by making a request to the servers at startup.
+If you would like to use this feature with the external Consul servers, you need to configure the Helm chart with the information about the server
+so that it can retrieve client's CA to use for securing the rest of the cluster.
+To do that, you must add the following values, in addition to the values mentioned above:
```

Comment:
```suggestion
If you would like to use this feature with external Consul servers, you need to configure the Helm chart with information about the servers
so that it can retrieve the clients' CA to use for securing the rest of the cluster.
To do that, you must add the following values, in addition to the values mentioned above:
```